sections/main-navbar.liquid

```diff
@@ -91,7 +91,7 @@
       {%- endcase %}
     {%- endfor -%}
   </nav>
-  <div class='navigation-drawer navbar-drawer block md:hidden'>
+  <div class='navigation-drawer navbar-drawer block lg:hidden'>
```

Comment:
remove tailwind css classes

Reply:
We will remove it on a separate PR and clean up all of the tailwind classes across all files